views/windows.tmpl

```diff
@@ -0,0 +1,93 @@
+{{define "windows"}}
+<!DOCTYPE html>
+<html lang="en">
+
+{{template "html-head" "Decred Blocks Windows"}}
+<body class="{{ theme }}">
+    {{template "navbar" . }}
+    <div class="container main" data-controller="main">
+        <h4><span>Windows</span></h4>
+        {{ $windowsCount := (int64 (len .Data)) }}
+        <div class="mb-1 fs13">
+            {{if gt $windowsCount 0}}
+            {{ $topWindow := (int64 (index .Data 0).Window) }}
+            <div class="d-flex flex-wrap-reverse align-items-center justify-content-end list-display">
+                <span class="fs12 nowrap text-right">
+                    {{intComma (add $.OffsetWindow 1)}} &mdash; {{intComma (add $.OffsetWindow $windowsCount)}} of {{ intComma (add .BestWindow 1) }} windows
+                </span>
+                <nav aria-label="blocks navigation" data-limit="{{.Limit}}" class="ml-2">
+                    <ul class="pagination mb-0 pagination-sm">
+                        <li class="page-item {{if le (index .Data (subtract $windowsCount 1)).Window 1}}disabled{{end}}">
```

Comment:
We need to be super careful with these `index` statements as it's a good way to cause a panic.  I didn't think through the logic here to know if the value `(subtract $windowsCount 1)` will ever be out of range of `.Data`, but we need to look closely.

Reply:
`{{if gt $windowsCount 0}}` This line takes care when the `.Data` is empty.